news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -0,0 +1,100 @@
+
+
+### Comment and vote for the cards
+
+When clicking on a card, you can see two fields at its bottom: one to comment and another one to vote. It is really important to the team as it helps us evaluate how the feature impacts **you**. Here is what happens behind the scenes:
+
+![PrestaShop Productboard user score](/assets/images/2020/05/productboard-user-score.png)
+
+When rating the impact of a feature, Productboard adds your vote to a “user score” that the team takes directly into account when evaluating features through its prioritization score.
+The more an idea is voted, the better rank it gets, and the more probability it has to integrate the software.
+
+This “user score” is not the only indicator that the team takes into account when evaluating a feature but it is key. Complexity, innovation, business impact for the merchants, legal aspects, are some of the criteria studied as well.
+
+<div class="alert alert-note" role="alert">
+Of course, the goal is not to vote “Critical” on every feature idea. This would cancel the benefits of the open votes.</div>
+
+Now you might also wonder what happens when you add a comment.
+Well, the comment is automatically associated with the card in the team interface.
+It helps the product team, the designers, and the developers to precisely **identify** the users’ needs, and thus **build** the feature the best way possible to answer these needs.
+
+In the end, it is all about delivering as much value as possible to the users, with all kinds of users in mind.
+
+### Submit a new idea
+
+In addition to having an overview of the selected features, commenting and voting, you can submit new ideas.
+
+A “Submit an idea” button is located in the portal’s top-right corner. Click on it and follow the steps.
+
+![How to sumbit a new idea on PrestaShop Feedback](/assets/images/2020/05/productboard-new-idea.gif)
+
+Here again you can indicate the importance of this insight for you.
+
+Emails addresses are required by default by Productboard to avoid spamming.
+The team might use it to contact you about your idea as well.
+
+
+## Know what is next
+
+Once the problem exposed, the solution defined and the design approved, the feature enters its development phase. For the Core product team, the next steps are writing specifications on the [dedicated repository](https://github.com/PrestaShop/prestashop-specs) and creating all related issues so that the Core dev team or the community can have all the necessary elements to develop them. All those steps are lead on GitHub, Productboard is just the tool that allows the Core product team to collect feedback and prioritize features.
```

Comment:
creating all related issues on Github

Reply:
```suggestion
Once the problem exposed, the solution defined and the design approved, the feature enters its development phase. For the Core product team, the next steps are writing specifications on the [dedicated repository](https://github.com/PrestaShop/prestashop-specs) and creating all related issues on GitHub so that the Core dev team or the community can have all the necessary elements to develop them. All those steps are lead on GitHub, Productboard is just the tool that allows the Core product team to collect feedback and prioritize features.
```